### ISSUE TITLE
Add copy and paste 

### ISSUE THREAD

**Issue by thegreenman** [zikalify/StoicReading#12]
First, 
Thank you for taking the time to create and maintain this app. 
This is a more of a feature request... I'd love the option of copying and pasting quotes from this app so I can share them elsewhere. 
Cheers

**Comment by zikalify** [comment]
Actually, the app used to be able to do that but I turned it off. Because the texts are hard coded into the app the strings can only be a certain length, on some of the longer texts like a handful of Seneca's letters I've had to split the text into two strings and have second one displayed after the first. When you clicked and held on the first string and press select all it wouldn't highlight the other string. At some point when I've worked out a themes settings page I'll probably look into reading texts from their own file rather than stashing everything in the strings.xml file (getting to be a bit of a mess). I'll see if anything can be done then to resolve copy and pasting.

**Comment by thegreenman** [comment]
Thanks. Appreciated.

**Comment by arnoudspammer** [comment]
Thank you for this app and your work on this. I was curious as well for copying. I'll wait in the meantime.

**Comment by TacoTheDank** [comment]
@zikalify You could put every text into a specific text file for each author in an assets folder, and have the app call those when displaying a certain text, I think. You would have each text inside their own asset file, which would be located in a folder in assets specifically for a certain author.

**Comment by zikalify** [comment]
Yeah, I read something along those lines too but I'd have to look into it.
